angrmanagement/ui/dialogs/preferences.py

```diff
@@ -203,7 +226,8 @@ def _init_widgets(self):
         fmt: str = Conf.log_timestamp_format
         ts = datetime.now()
         # pylint: disable=use-sequence-for-iteration
-        self._fmt_map = bidict({ts.strftime(i): i for i in {fmt, "%X", "%c"}})  # set also dedups
+        self._fmt_map = bidict({ts.strftime(i): i for i in ("%X", "%c")})
```

Comment:
Should probably be in a different PR

Reply:
This is actually a bug fix that prevents duplicate keys from appearing in the initial dictionary, which previously might remove the presently installed format from the dict. That's why it's now inserted after the fact to ensure that it exists; this has the side effect that other formats in that initial set that map to the exact same timestamp when formatted will not appear as default options, but that's fine. (I.e. if "%A" and "%B" are different format strings that yield the same output, this `bidict` cannot handle it so the presently installed one is preferred over any 'default options')

Reply:
Lacking this fix, an exception will be raised on launch.